python-clusters/create-eks-cluster/cluster.json

```diff
@@ -41,9 +41,9 @@
             "label": "Cluster nodes"
         },
         {
-            "name": "nodePool",
-            "label": "Initial node pool",
-            "type": "PRESET",
+            "name": "nodePools",
+            "label": "Node pools to create in the cluster",
+            "type": "PRESETS",
```

Comment:
The `PRESETS` type does not seem to allow for discarding configs. They can be set to None, but not entirely removed. Is it worth logging a card for that?